<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>showpicture</class>
 <widget class="QDialog" name="showpicture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="headLabel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Photos</string>
   </property>
  </widget>
  <widget class="QLabel" name="showPicture">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>590</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="pictureName">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>550</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
